src/test/java/com/systelab/seed/patient/BodyMassIndexCalculatorTest.java

```diff
@@ -0,0 +1,74 @@
+package com.systelab.seed.patient;
```

Comment:
Change the package to be the same as the source code.
```suggestion
package  com.systelab.seed.patient.control;
```